reshade.example.ini

```diff
@@ -0,0 +1,18 @@
+[APP]
+ForceDefaultRefreshRate=0
+ForceFullscreen=0
+ForceVsync=0
+ForceWindowed=0
+
+[GENERAL]
+EffectSearchPaths=.\reshade-shaders\Shaders,.\reshade-shaders\Shaders\qUINT,.\reshade-shaders\Shaders\PD80,.\reshade-shaders\Shaders\AstrayFX,.\gshade-shaders\Shaders,.\gshade-shaders\ComputeShaders
```

Comment:
We may not want to by default allow both `reshade-shaders` and `gshade-shaders`. If someone ends up with both directories in their area for whatever reason shaders will double load which is bad. Same story for textures.